ml/jml/onevsall.h

```diff
@@ -0,0 +1,72 @@
+// This file is part of MLDB. Copyright 2017 mldb.ai inc. All rights reserved.
+
+/* onevsall.h                                          -*- C++ -*-
+   Mathieu Marquis Bolduc, 8 March 2017
+   Copyright (c) 2017 MLDB.ai  All rights reserved.
+   $Source$
+
+   onevsall classifier
+*/
+
+#pragma once
+
+
+#include "mldb/ml/jml/classifier.h"
+#include "feature_set.h"
+#include "mldb/ml/jml/feature_map.h"
+
+
+namespace ML {
+
+
+/*****************************************************************************/
+/* ONEVSALL_CLASSIFIER                                                       */
+/*****************************************************************************/
+
+/** Classifier that always returns the zero label. */
```

Comment:
I don't understand the design here.  Could you add a comment that describes what we're attempting to do with this classifier?